src/coreclr/tools/InjectResource/Program.cs

```diff
@@ -1,12 +1,12 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
+// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 using System.CommandLine;
 using System.IO;
 
-var binOption = new Option<FileInfo?>(
+var binOption = new Option<FileInfo>(
```

Comment:
@jkotas, these were failing with nullability warnings on some legs (with `./build.sh mono` but not with `./build.sh clr`).